src/python/bot/fuzzers/syzkaller/engine.py

```diff
@@ -79,10 +79,9 @@ def prepare(self, corpus_dir, target_path, unused_build_dir):  # pylint: disable
     Returns:
       A FuzzOptions object."""
     self.prepare_binary_path()
-    config = runner.get_config()
     return SyzkallerOptions(
         corpus_dir,
-        config,
+        runner.get_cover_file_path(),
```

Comment:
I think you meant runner.get_config(), but good to just revert to local var as before.